### ISSUE TITLE
Regarding the bap recipe installation

### ISSUE THREAD

**Issue by anzosasuke** [BinaryAnalysisPlatform/bap-toolkit#13]
Trying to install bap-toolkit recipes.
Ocaml version: 4.05

``ocamlfind: Package 'bap-taint' not found, error ``

**Comment by ivg** [comment]
You need to [install bap][1] first, then make sure that opam installation is activated with `eval $(opam env)`. There is also a prebuilt docker image with pre-built toolkit, and see also the dockerfile and readme on how to use it. Feel free to reopen if you still have problems!

[1]: https://github.com/BinaryAnalysisPlatform/bap#installation

**Comment by anzosasuke** [comment]
I have it installed. i tried bap2.5.0, 2.4.0, 2.3.0 but still got this errors

**Comment by ivg** [comment]
I don't think you have it installed (at least because OCaml 4.05 is so ancient version that I don't remember when we dropped it). What does `bap --version` show?

**Comment by ivg** [comment]
Btw, could it be that you have installed it through a debian package? If yes then you can't build bap-toolkit with a debian version, because it just installs the binary, but not the development environment and the necessary toolchain, you have to install bap from opam, following the instructions I referenced above.

**Comment by anzosasuke** [comment]
Yeah. That was the case. but used dockerized version of it. Is dockerized version any different?

**Comment by ivg** [comment]
> Yeah. That was the case. but used dockerized version of it. Is dockerized version any different?

Which dockerized version are you referring to? You can use `binaryanalysisplatform/bap` as your base image to build anything that requires bap, e.g.,
```
FROM binaryanalysisplatform/bap:latest
```

See the toolkit's [Dockerfile][1] for the reference. 

[1]: https://github.com/BinaryAnalysisPlatform/bap-toolkit/blob/master/Dockerfile

**Comment by anzosasuke** [comment]
The same one the dockerfile mentioned. I built it using that dockerfile.
